include/gmock-win32.h

```diff
@@ -64,7 +65,7 @@ namespace detail {
     {
         static Derived& instance()
         {
-            static ::testing::NiceMock< Derived > obj;
+            static ::testing::template NiceMock< Derived > obj;
```

Comment:
Fix for C++20 build

Reply:
I've never seen anything like this, very interesting.

Reply:
> I've never seen anything like this, very interesting.

also never seen that )